src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -210,8 +213,20 @@ create_gtod()
 
     chmod 0755 ./gtod
 
-    TIME_ZERO=`./gtod`
-    export TIME_ZERO
+    # initialize profile
+    PROFILE="bootstrap_0.prof"
+    now=$(./gtod)
+    echo "#time,name,uid,state,event,msg" > "$PROFILE"
```

Comment:
update as `#time,event,comp,thread,uid,state,msg`?

Reply:
Woah - good catch, that line must have quite a history of blind copy-and-paste... :-P